src/main/java/prefs/Preferences.java

```diff
@@ -134,36 +280,54 @@ public void setLastOpenBoard(String board) {
             int index = (lastBoard + 1) % boardNames.size();
             
             setLastOpenBoard(boardNames.get(index));
+            save();
```

Comment:
I think I asked you to move it here, my bad :/ Since `setLastOpenBoard` already calls `save`, I don't think we need `save` here.